lib/device/library/linux.rb

```diff
@@ -2,11 +2,14 @@
 #
 # Copyright 2013 whiteleaf. All rights reserved.
 #
+require 'etc'
```

Comment:
__[rubocop]__
* Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

<!-- Comment by [SideCI](https://sideci.com) -->